backend/app/Http/Controllers/UserController.php

```diff
@@ -2,13 +2,38 @@
 
 namespace App\Http\Controllers;
 
+use App\Http\Resources\LoginResource;
 use App\Http\Resources\UserResource;
 use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Validation\Rule;
+use Illuminate\Validation\ValidationException;
 
 class UserController extends Controller
 {
     public function index()
     {
         return UserResource::collection(User::all());
     }
+
+    public function login(Request $request)
+    {
+        $credentials = $this->validate($request, [
+            'email' => ['required', Rule::exists('users', 'email')],
+            'password' => ['required'],
+        ]);
+
+        if (! Auth::attempt($credentials)) {
+            throw ValidationException::withMessages([
+                'email' => 'Your provided credentials could not be verified.',
```

Comment:
Just a question, when will this appear? I tried logging in with wrong credentials but still did not appear.